packages/vuelidate/src/core.js

```diff
@@ -150,19 +143,18 @@ function createAsyncResult (rule, model, initResult, $pending) {
  * @return {{$params: *, $message: Ref<String>, $pending: Ref<Boolean>, $invalid: Ref<Boolean>}}
  */
 function createValidatorResult (rule, state, key) {
-  const model = computed(() => state[key].value)
-  const ruleResult = callRule(rule.$validator, model)
+  const ruleResult = callRule(rule.$validator, state[key])
```

Comment:
We can get rid of this part with the lazy validations I implemented. Do you think it could be further improved with the changes that you made now?

Reply:
Correct. Have not synced yet.